docs/lifecycle.md

```diff
@@ -2,52 +2,47 @@
 
 Life cycle assessment or LCA (also known as life cycle analysis) is a methodology for assessing environmental impacts associated with all the stages of the life cycle of a commercial product, process, or service. For instance, in the case of a manufactured product, environmental impacts are assessed from raw material extraction and processing (cradle), through the product's manufacture, distribution and use, to the recycling or final disposal of the materials composing it (grave). (from ["Life-Cycle Assessment"](https://en.wikipedia.org/wiki/Life-cycle_assessment) - Wikipedia).
 
-From [Wikipedia](https://en.wikipedia.org/wiki/File:Life_cycle_analysis_and_GHG_carbon_accounting.jpg), an LCA diagram with GHG phases along the right side
-![Example of an LCA with GHG emissions](images/lca_ghg.jpeg)
-
 ## Life Cycle Assessment of an Advertisement
 
 To "manufacture" an ad impression, our raw materials are media (content), creative (the ad itself), and human attention (via ears and/or eyes). Different channels will have different combinations of these ingredients. For instance, a billboard often doesn't have content, just the ad. However, a billboard will have a disposal component when the vinyl is removed or recycled.
 
-The life cycle of an ad has five major components:
+The life cycle of an ad can be broken down into two stages: production and use. The production phase includes media production and creative production. The use phase has four components:
 
-1. Producing the creative
-2. Producing the content
-3. Selecting which ad to display
-4. Consumption of the media and creative, including disposal
-5. Consumer behavior after consuming the ad
+1. Media distribution
+1. Ad selection
+1. Consumer engagement - the impression
+1. Disposal
```

Comment:
and in parens `(Ad disposal and media disposal)`? just to be very clear the connection between this list and image below.

also, would be cool if the image could somehow have something to mark the sections connected to use and the sections connected to production